models/customer/src/transformers.ts

```diff
@@ -1,15 +1,36 @@
 import { Transformer } from '@commercetools-test-data/core';
 import type { TCustomer, TCustomerGraphql } from './types';
 
+const commonBuildFields: (keyof TCustomer)[] = [
+  'createdBy',
+  'lastModifiedBy',
+  'stores',
+  'addresses',
+];
+
+const removeFields: (keyof TCustomer)[] = [
```

Comment:
Since these ones are not shared and we use them only in the `rest` transformer, would it be more easy to follow if we just set them inline?

Reply:
https://github.com/commercetools/test-data/pull/634/commits/2c0abbc1ff15aa5aa7282cee028c4bdbcd8b9339